test/rational.jl

```diff
@@ -265,6 +265,92 @@ end
         @test read(io2, typeof(rational2)) == rational2
     end
 end
+@testset "parse" begin
+    # Non-negative Int in which parsing is expected to work
+    @test parse(Rational{Int}, string(Int(10))) == Int(10) // Int(1)
+    @test parse(Rational{Int}, "100/10" ) == Int(10) // Int(1)
+    @test parse(Rational{Int}, "$(Int(100))/$(Int(10))") == Int(10) // Int(1)
```

Comment:
You are correct. These existed because the same was not true for the other types and the `for` loop that generated the tests did generate them for all types without discrimination. I forgot to remove them when I removed the tests for the other types.

Reply:
Fixed in the last commit.